### ISSUE TITLE
Upgrade guide for upgrading pd from 2.0.x to 2.1.0

### ISSUE THREAD

**Issue by Guru107** [pingcap/pd#1373]
Please answer these questions before submitting your issue. Thanks!
Is there an upgrade guide for upgrading pd to 2.10? Just doing a rolling upgrade work?
1. What did you do?
If possible, provide a recipe for reproducing the error.

2. What did you expect to see?

3. What did you see instead?

4. What version of PD are you using (`pd-server -V`)?

**Comment by disksing** [comment]
Hi, @Guru107 . Under normal circumstances rolling update will work, but there are some conditions need to check to avoid potential problems. Please check the release document of v2.1.0: https://pingcap.com/docs/releases/2.1ga/#upgrade-caveat

**Comment by Guru107** [comment]
@disksing The link that you posted, is it related to TiDB specific or applies to PD as well?

**Comment by disksing** [comment]
Yes. That is the generalized `TiDB` which includes all TiDB cluster components (tidb-server, tikv-server, pd-server).
